install/debian/postinst.in

```diff
@@ -1,59 +1,64 @@
 #!/bin/sh -e
 
 NAME=fah-client
+CLIENT_CONFIG=/etc/$NAME
 CLIENT_HOME=/var/lib/$NAME
 CLIENT_LOGS=/var/log/$NAME
 
-case "$1" in
-  reconfigure|configure)
-    # Create group if it does not exist
-    if ! getent group $NAME >/dev/null; then
-      groupadd --force --system $NAME
-    fi
-
-    # Create user if it does not exist
-    if ! getent passwd $NAME >/dev/null; then
-      useradd --system --gid $NAME --shell /usr/sbin/nologin \
-        --home-dir $CLIENT_HOME --no-create-home --groups video,render $NAME
-    fi
-
-    # Create directories
-    install -d -m 755 -o $NAME -g $NAME $CLIENT_HOME $CLIENT_LOGS
-
-    # Create a symbolic link to the configuration file
-    ln -sf /etc/fahclient/config.xml $CLIENT_HOME/config.xml
-
-    # Remove old unit if it exists
-    rm -f /etc/systemd/system/${NAME}.service
-
-    # Start the service
-    if [ -n "$2" ]; then
-      ACTION=restart
-    else
-      ACTION=start
-    fi
-    systemctl daemon-reload
-    systemctl enable $NAME
-    systemctl $ACTION $NAME
+# Create directories
+install -d -m 755 -o $NAME -g $NAME $CLIENT_HOME $CLIENT_LOGS
 
+# migrate configuration
+
+# remove obsolete symlink installed by previous v8 packages
+[ -L $CLIENT_HOME/config.xml ] && rm -f $CLIENT_HOME/config.xml
```

Comment:
This will fail the script if `$CLIENT_HOME/config.xml` is *not* a symlink. Needs `|| true` at the end.

Reply:
Fixed, thanks